etc/yum-cron-hourly.conf

```diff
@@ -20,6 +20,10 @@ download_updates = no
 # the update to be applied
 apply_updates = no
 
+# You may set days_of_week to the numeric days of the week you want to run,
+# where 0 is Sunday and 6 is Saturday. The default is to run every day.
```

Comment:
date.weekday() starts on Monday so Sunday is 6 (not 0), see:
https://docs.python.org/2/library/datetime.html#datetime.date.weekday

Reply:
Sorry for the delay. I'm not sure how i'd like to handle this. Should we break backwards compatibility with yum-cron that ships with el6, or refactor so 0 is Sunday. 

I'm leaning on updating the docs, and breaking the precedent. What do you all think?